ci-operator/templates/openshift/openshift-azure/build-node-image.yaml

```diff
@@ -231,7 +231,8 @@ objects:
         cd /go/src/github.com/openshift
         git clone https://github.com/openshift/openshift-azure
         cd openshift-azure/
-
+        mkdir -p /go/src/github.com/openshift/openshift-azure/secrets
+        cp /etc/azure/credentials/*.cert /etc/azure/credentials/*.key /etc/azure/credentials/*.key /etc/azure/credentials/.dockerconfigjson /go/src/github.com/openshift/openshift-azure/secrets/
```

Comment:
Please pass the names verbatim.

Reply:
there will be 2 more files under the same pattern soon. This will get very long very soon.